### ISSUE TITLE
False positive on empty_line_after_outer_attr

### ISSUE THREAD

**Issue by topecongiro** [rust-lang-nursery/rust-clippy#2517]
When you run `cargo clippy` against the following code,

```rust
#[must_use]
#[derive(Debug, Default, Clone, Copy)]
pub struct Summary {
    // Encountered e.g. an IO error.
    has_operational_errors: bool,

    // Failed to reformat code because of parsing errors.
    has_parsing_errors: bool,

    // Code is valid, but it is impossible to format it properly.
    has_formatting_errors: bool,

    // Formatted code differs from existing code (write-mode diff only).
    pub has_diff: bool,
}
```

you will get the following warning:

```
warning: Found an empty line after an outer attribute. Perhaps you forgot to add a '!' to make it an inner attribute?
 --> src/lib.rs:3:1
  |
3 | pub struct Summary {
  | ^
  |
  = note: #[warn(empty_line_after_outer_attr)] on by default
  = help: for further information visit https://rust-lang-nursery.github.io/rust-clippy/v0.0.187/index.html#empty_line_after_outer_attr
```

Using `0.0.187`.

**Comment by oli-obk** [comment]
cc @phansch

**Comment by phansch** [comment]
Looks like the automatic close didn't work on this one. This was fixed by #2577 and is in 0.0.190

**Comment by topecongiro** [comment]
@phansch Thanks for your work! FWIW you need to write the verb before every issue number, like `Fixes #2475. Fixes #2517.` to make automatic closing work.
